README.md

```diff
@@ -77,8 +77,8 @@ function is_expired(args, tuple)
   return true
```

Comment:
Isn't it a time to create additional `space` input option and add a log warning that `space_id` is deprecated?

Reply:
I'm sorry, but I didn't get it.

`space` looks good as a required parameter, not option.

Reply:
Oh, it looks like `space`/`space_id` is a positional argument in Lua-style task.

```Lua
expirationd.start(job_name, space.id, is_expired, opts)
```

I have thought that it is a named one.